static/css/index.css

```diff
@@ -22,9 +22,8 @@
 body {
   font-size: 16px;
   font-family: "Poppins", sans-serif;
-  font-weight: 400px;
   line-height: 1.428;
-  background-color: #e5e5e5;
+  background-color: #f7f7f9;
 }
 
 ::-webkit-scrollbar {
```

Comment:
- [ ] Kindly make sure your page is not overflowing on the right as you can see on these images.
![Screenshot from 2022-09-23 00-24-07](https://user-images.githubusercontent.com/97100091/191863015-17c464d0-4c56-4051-a91f-c1b1258136d3.png)
![Screenshot from 2022-09-23 00-23-42](https://user-images.githubusercontent.com/97100091/191863056-64add8ce-c82f-40d1-8610-f7ddacea8aad.png)

- [ ] Kindly consider giving your `images` a `width` in `percentage` and a `height` `auto` to make them responsive

Reply:
- Thanks @christianbiring1 🙏 for reviewing my pull request I appreciate your review 💯 
### Thanks again for reviewing my PR  and have a good day😊